src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/ConfirmOutcome.cshtml

```diff
@@ -0,0 +1,216 @@
+@using SFA.DAS.AssessorService.Domain.Extensions;
+@using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+
+@model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.RoatpGatewayApplicationViewModel
+
+@{
+    ViewBag.Title = "Gateway - Overall gateway outcome";
+    Layout = "_Layout";
+}
+
+@{
+    var formGroupError = Model.ErrorMessages != null && Model.ErrorMessages.Any() ? "govuk-form-group--error" : "";
+    var radioCheckedAskClarification = Model.GatewayReviewStatus == "Ask Clarification" ? "checked" : "";
+    var radioCheckedDeclined = Model.GatewayReviewStatus == "Declined" ? "checked" : "";
+    var radioCheckedApproved = Model.GatewayReviewStatus == "Approved" ? "checked" : "";
+
+    var isGatewayReviewStatus = false;
+    var isErrorAskClarification = false;
+    var isErrorDeclined = false;
+    var isErrorApproved = false;
+
+    var isErrorAskClarificationCss = "";
+    var isErrorDeclinedCss = "";
+    var isErrorApprovedCss = "";
+
+    var ErrorGatewayReviewStatus = "";
+    var ErrorAskClarificationText = "";
+    var ErrorDeclinedText = "";
+    var ErrorApprovedText = "";
```

Comment:
Can some of this be done in the viewmodel?

Reply:
I agree, it should be in the view model and unit tested.